tests/git_bisect_toaster.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# These variables are fully defined in runtests.sh
+# Just short descripstions appear here
+# what container to run. Usually crops/toaster-master
+export IMAGE=crops/toaster-master
```

Comment:
Why not leave this as the default from runtests.sh, "crops/toaster:latest"? Since it's using a local poky as toaster, the IMAGE for the container really shouldn't affect anything.